nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -132,12 +128,11 @@ def setup_experiment(self, config):
                                  passed to the constructor
             - loss_function: Loss function. See "torch.nn.functional"
             - local_dir: Results path
+            - logdir: Directory generated by Ray Tune for this Trial
             - epochs: Number of epochs to train
             - batches_in_epoch: Number of batches per epoch.
                                 Useful for debugging
             - progress: Show progress during training
-            - profile: Whether or not to enable torch.autograd.profiler.profile
-                       during training
```

Comment:
You need to remove this option from [run.py](https://github.com/numenta/nupic.research/blob/master/projects/imagenet/run.py#L62-L63) as well

Reply:
Nice catch. I'll do something even better and allow you to add the profile mixins via `--profile` and `--profile-autograd`